content/zh/docs/reference/glossary/kube-controller-manager.md

```diff
@@ -31,10 +31,13 @@ tags:
 <!--
  Control plane component that runs {{< glossary_tooltip text="controller" term_id="controller" >}} processes.
 -->
-运行{{< glossary_tooltip text="控制器" term_id="controller" >}}进程的控制平面组件。
+`kube-controller-manager` 是{{< glossary_tooltip text="控制平面" term_id="control-plane" >}}的组件，
+负责运行{{< glossary_tooltip text="控制器" term_id="controller" >}}进程。
 
-<!--more--> 
```

Comment:
The [preview](https://deploy-preview-34113--kubernetes-io-main-staging.netlify.app/zh/docs/reference/glossary/?fundamental=true&operation=true#term-kube-controller-manager) is broken. :(